server/services/locationGroupService.ts

```diff
@@ -1,13 +1,13 @@
 import { RestClientBuilder, VisitSchedulerApiClient, HmppsAuthClient } from '../data'
-import { SessionLocationGroup } from '../data/visitSchedulerApiTypes'
+import { LocationGroup } from '../data/visitSchedulerApiTypes'
 
-export default class LocationGroupsService {
+export default class LocationGroupService {
```

Comment:
I've renamed this service to be `LocationGroup` (singular) for consistency with existing Prison and SessionTemplate ones. Singular because it's the 'thing' it's dealing with and it may handle individual or multiple ones.